src/net.cpp

```diff
@@ -135,11 +135,11 @@ static std::vector<CAddress> convertSeed6(const std::vector<SeedSpec6> &vSeedsIn
     const int64_t nOneWeek = 7*24*60*60;
     std::vector<CAddress> vSeedsOut;
     vSeedsOut.reserve(vSeedsIn.size());
-    for (std::vector<SeedSpec6>::const_iterator i(vSeedsIn.begin()); i != vSeedsIn.end(); ++i)
+    for (const auto seed_in : vSeedsIn)
```

Comment:
`const SeedSpec6& seed_in` makes more sense to me as it prevents a copy, no?

Reply:
Correct. Done!